### PR TITLE
Fix manga downloader

### PR DESCRIPTION
**Pull request by flamura**
:D

### CODE REVIEW

**Comment by akameco** [comment]
Thank you for PR @trnhoe :)

File: index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = (tag, opts) => {
 };
 
 function * main(tag, opts) {
-	const pixiv = new Pixiv(opts.username, opts.password);
+	const pixiv = new Pixiv(opts.username.toString(), opts.password.toString());
```
Comment (akameco) [inline index.js:28]: Why `toString()`?

Comment (flamura) [reply index.js:28]: My password is a number,while `Meow` wont convert it to a string which `PixivAuthGot` needs.

Comment (akameco) [reply index.js:28]: Oh, I see.

**Pull request merged**